services/upload/pinata.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+import axios from 'axios'
+
+const endpoint = 'https://api.pinata.cloud/pinning/pinFileToIPFS'
+
+export const uploadToPinata = async (
+  fileArray: File[],
+  pinataApiKey: string,
+  pinataSecretKey: string,
+  fileType?: string,
+): Promise<string> => {
+  console.log('Uploading to Pinata...')
+  if (fileType && fileType === 'metadata') {
```

Comment:
Instead of using if else in here and duplicating code, let's assign this image prefix to a variable and use it that way. If we make use of the created type above;

```suggestion
data.append('file', file, `${fileType}/${file.name}`)
```